rotkehlchen/chain/ethereum/modules/curve/decoder.py

```diff
@@ -59,7 +76,7 @@ def _decode_curve_remove_events(
             transaction: EvmTransaction,
             tx_log: EvmTxReceiptLog,
             decoded_events: list[HistoryBaseEntry],
-            user_address: ChecksumEvmAddress,
+            maybe_user_address: ChecksumEvmAddress,
     ) -> tuple[Optional[HistoryBaseEntry], list[ActionItem]]:
         """Decode information related to withdrawing assets from curve pools"""
```

Comment:
Can you add a comment in this function to explain why it's `maybe_` prefixed? So it is users address or ... what? In which case a better name may be `user_or_pool_address`? Or I am not understanding the logic here?